video_loader.py

```diff
@@ -178,6 +204,23 @@ def _convert_video_uri_to_audio(self, source_fn, uri, ffmpeg_args, librosa_args)
         finally:
             return data, sample_rate
 
+    def _convert_video_uri_to_subtitle(self, source_fn, uri, ffmpeg_args, tmp_dir):
```

Comment:
If `uri` is only used for logging, I don't see why we need to keep it.
```suggestion
    def _convert_video_uri_to_subtitle(self, source_fn, ffmpeg_args, tmp_dir):
```